.github/workflows/run-unit-tests.yml

```diff
@@ -93,7 +93,7 @@ jobs:
             colima start --network-address --arch x86_64 --cpu=1 --memory=1
           }
 
-          for i in $(seq 1 $ATTEMPT_LIMIT); do
+          for ((i=1; i<=ATTEMPT_LIMIT; i++)); do
```

Comment:
How does this work differently than before? 🤔

Reply:
So the following will work
https://github.com/OpenDevin/OpenDevin/blob/5a5713009f7a0b3fd2f61f43bf14452bf1b92ad5/.github/workflows/run-unit-tests.yml#L105

Reply:
Ok, worth a try.

Lets also fix another thing while we are in the same file, please change this line:
https://github.com/OpenDevin/OpenDevin/blob/e887b84a6cd3efc27559e57e0f57cecf4bef06af/.github/workflows/run-unit-tests.yml#L123

To also include **` and not test_runtime`** in the conditions (like the Linux call has it):
`        run: poetry run pytest --forked --cov=agenthub --cov=opendevin --cov-report=xml ./tests/unit -k "not test_sandbox and not test_runtime"`

Reply:
reason?

Reply:
Because it was forgotten in the original PR (nothing to do with you)

Reply:
Applying for https://github.com/OpenDevin/OpenDevin/issues/2992

Reply:
I can start a PR for this quick fix - it was my fault though :(

Reply:
did